apps/builddao/widget/components/project/page/Discussion.jsx

```diff
@@ -1 +1,69 @@
-return <div className="text-white">Discussion</div>;
+const { Feed } = VM.require("${alias_devs}/widget/Feed") ?? {
```

Comment:
Let's rename this "Activity" rather than "Discussions" and then let's get this merged. We'll scope out "Discussions" and gather requirements in a separate ticket